tools/benchmark/src/main/java/com/datadog/benchmark/DatadogExporterConfiguration.kt

```diff
@@ -0,0 +1,148 @@
+    val resource: String? = null,
+    val applicationName: String? = null,
+    val applicationVersion: String? = null,
+    val applicationId: String? = null,
+    val apiKey: String,
+    val run: String? = null,
+    val endPoint: EndPoint = EndPoint.US1,
+    val intervalInSeconds: Long = DEFAULT_INTERVAL_IN_SECONDS
+) {
+    /**
+     * A Builder class for a [DatadogExporterConfiguration].
+     *
+     * @param apiKey api key to submit metrics to Datadog endpoint
+     */
+    class Builder(private val apiKey: String) {
+        private var serviceName: String? = null
+        private var resource: String? = null
+        private var applicationName: String? = null
+        private var applicationVersion: String? = null
+        private var applicationId: String? = null
+        private var run: String? = null
+        private var endPoint: EndPoint = EndPoint.US1
+        private var intervalInSeconds: Long = DEFAULT_INTERVAL_IN_SECONDS
+
+        /**
+         * Sets the service name for the metric.
+         *
+         * @param serviceName service name used as resource service name in metric data
+         */
+        fun setServiceName(serviceName: String): Builder {
+            this.serviceName = serviceName
+            return this
+        }
+
+        /**
+         * Sets the resource name for the metric.
+         *
+         * @param resource resource name
+         */
+        fun setResource(resource: String): Builder {
+            this.resource = resource
+            return this
+        }
+
+        /**
+         * Sets the host application name for the metric.
+         *
+         * @param applicationName application name
+         */
+        fun setApplicationName(applicationName: String): Builder {
+            this.applicationName = applicationName
+            return this
+        }
+
+        /**
+         * Sets the interval of meter sampling and upload.
+         *
+         * @param intervalInSeconds interval in seconds
+         */
+        fun setIntervalInSeconds(intervalInSeconds: Long): Builder {
+            this.intervalInSeconds = intervalInSeconds
+            return this
+        }
+
+        /**
+         * Sets the host application id, default value is 10 seconds.
+         *
+         * @param applicationId host application id
+         */
+        fun setApplicationId(applicationId: String): Builder {
+            this.applicationId = applicationId
+            return this
+        }
+
+        /**
+         * Sets the host application version name.
+         *
+         * @param applicationVersion application version name
+         */
+        fun setApplicationVersion(applicationVersion: String): Builder {
+            this.applicationVersion = applicationVersion
+            return this
+        }
+
+        /**
+         * Sets the run environment description.
+         *
+         * @param run run environment description
+         */
+        fun setRun(run: String): Builder {
```

Comment:
why not call it `setEnv(env: String)` then?

Reply:
Because in the RFC the tag name should be `run`, so here I want to align with it:
``` 
[
  "device_model:iphone1.2",
  "os:ios",
  "os_version:17.4",
  "run:baseline|instrumented|profiling",
  "scenario:sr|rum|trace|logs",
  "application_id:com.datadoghq.foo",
  "sdk_version:2.0.0",
  "branch:develop"
]

```